src/main/java/wiggly/commands/AddCommand.java

```diff
@@ -0,0 +1,75 @@
+package wiggly.commands;
+
+import java.time.LocalDate;
+
+import wiggly.task.TaskList;
+import wiggly.task.TaskType;
+
+import wiggly.task.Task;
+import wiggly.task.ToDo;
+import wiggly.task.Deadline;
+import wiggly.task.Event;
+
+import wiggly.util.Storage;
+import wiggly.util.Ui;
+
+/**
+ * A class representation of the add command
+ */
+public class AddCommand extends Command {
+
+	private final TaskType taskType;
+	private final String description;
+	private final LocalDate by;
+	private final LocalDate from;
+	private final LocalDate to;
+
+	/**
+	 * Creates a {@code AddCommand} instance with the given taskType, description, and dates. If there is no date for
+	 * that parameter, use {@code null} instead.
+	 * @param taskType The type of task to instantiate
+	 * @param description The description of the task
+	 * @param by The due date of the task, if empty, use {@code null} as the parameter
+	 * @param from The start date of the task, if empty, use {@code null} as the parameter
+	 * @param to The end date of the task, if empty, use {@code null} as the parameter
+	 */
```

Comment:
You could add a blank line in the javadocs after description of method, before params for more readability